.travis.yml

```diff
@@ -20,63 +27,71 @@ before_install:
     -out keys.tgz
     -d
   
-  #Extract secrets
+  # Extract secrets
   - tar xzvf keys.tgz
   
-  #Add the keypair to the agent
+  # Add the keypair to the agent
   - eval "$(ssh-agent -s)"
   - ssh-add keys/id_rsa
   
-  #Get secrets
+  # Get secrets
   - git clone git@github.com:mcapuccini/secrets-kubenow.git secrets-kubenow
 
-install:
-  #INSTALL DEPS
+  # CHECK CODE QUALITY
+  # Installing Terraform and check with fmt command
+  - >
+      travis_retry curl
+      https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+      > /tmp/terraform.zip
+  - unzip /tmp/terraform.zip -d /tmp/
+  - /tmp/terraform fmt bin/gce_main.tf
+  - git diff --exit-code # this will fail if terraform changed something
+  
+  # Check Json
+  - python -mjson.tool < build-"$HOST_CLOUD".json
```

Comment:
This always return 0. You need to do the trick with `git diff --exit-code` like above:

*untested*
```
echo "$(python -mjson.tool < build-aws.json)" > build-aws.json 
git diff --exit-code
```